CHANGELOG.md

```diff
@@ -1,5 +1,31 @@
 # Changelog - v3
 
+## [v3.4.8] (May 19 2023)
+
+Fixes:
+* Prevent white space only text sending
+* Mentioned user Regex parsing
+  Mention will now work even if userId has `.*+?^${}()|[\]\\` characters
+* ChannelList blink when when message is send
+  Happened when there were two channelLists in the same page with
+  different query params
+* ChannelSetting `renderUserProfile` prop
+  We were applying `renderChannelProfile` in place of `renderUserProfile`
+* MessageBody: Words break mid word
+  Words were breaking midword because all white spaces
+  were converted into nbsps. CSS couldnt distinguish nbsps
+  as whitespaces, so wrapping didnt work well
```

Comment:
How about?
```suggestion
  Words were breaking in the middle because all white spaces were converted into nbsps. CSS couldn't differentiate nbsps as regular white spaces, resulting in poor text wrapping.
```